db/schema.rb

```diff
@@ -10,8 +10,50 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 0) do
+ActiveRecord::Schema[7.0].define(version: 2023_05_02_211542) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "comments", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "author_id", null: false
+    t.integer "post_id", null: false
+    t.text "text"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.bigint "post_id_id", null: false
+    t.bigint "author_id_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_id_id"], name: "index_likes_on_author_id_id"
+    t.index ["post_id_id"], name: "index_likes_on_post_id_id"
+  end
+
```

Comment:
- [ ] Kindly note that these should be `post_id` and `author_id`, not `post_id_id` and `author_id_id`. Kindly fix the errors.